src/app/Http/Controllers/Operations/ListOperation.php

```diff
@@ -71,9 +71,6 @@ public function search()
     {
         $this->crud->hasAccessOrFail('list');
 
-        // how many total entries there is applying the controller/model constrains, before any filtering or search
-        $this->crud->setOperationSetting('totalEntryCount', $this->crud->getTotalEntryCount());
```

Comment:
I don't think we can move this after the `applyUnappliedFilters` otherwise the "total entry count" would be the filtered results. 

This is the reason: https://github.com/Laravel-Backpack/CRUD/blob/074e4aaed590b7f96d8119bccf8c8bee4fa075fc/src/app/Library/CrudPanel/CrudFilter.php#L105

Anyway you think we can solve this ? Filters have the "current logic", "fallback logic" and "default logic", so even if you are not "filtering on table", some filters may be applied first.

That's the reason we need the total entry count to be set BEFORE any filters are applied. 

ping @tabacitu